yogstation/code/game/objects/items/fishing/bait.dm

```diff
@@ -30,20 +30,46 @@
 	if(prob(1))
 		desc = "Don't you dare say it"
 
+/*/obj/item/reagent_containers/food/snacks/bait/empowered //needed for the future, commented out for now
+	name = "empowered bait"
+	desc = "Bait infused with the power of Bluespace Slimes. What could this catch?..."
+	icon_state = "bait_3"
+	fishing_power = 100
+*/
+
 /obj/item/reagent_containers/food/snacks/bait/worm
 	name = "worm bait"
 	desc = "Might be able to catch a lizard with this..."
 	icon_state = "bait_worm"
 	fishing_power = 15
 
-/obj/item/reagent_containers/food/snacks/bait/leech
+/obj/item/reagent_containers/food/snacks/bait/worm/leech //subtype of worm so recipes work for bait
 	name = "leech"
 	desc = "What isn't fun about a little recycling?"
 	icon_state = "leech"
 	fishing_power = 20
 
+/obj/item/reagent_containers/food/snacks/bait/worm/leech/attack(mob/living/M, mob/living/user)
+	if(user.a_intent == INTENT_HARM)
+		return ..()//wait no not there this is hitting 
+	M.visible_message(span_danger("[user] is putting a leech onto [M]!"))
+	if(!do_after(user, 2 SECONDS, M)) 
+		M.visible_message(span_danger("[M] avoids the leech!"))
+		return FALSE
+	M.adjustToxLoss(-5)
+	M.blood_volume -= 5
+	for(var/datum/reagent/T in M.reagents.reagent_list)
+		if(istype(T, /datum/reagent/toxin))
+			M.reagents.trans_id_to(src, T.type, 5)
+	return FALSE 
+
 /obj/item/reagent_containers/food/snacks/bait/type
 	name = "type bait"
 	desc = "Are you talking to me?"
 	icon_state = "bait_t"
 	fishing_power = 25
+
+/obj/item/reagent_containers/food/snacks/bait/type/wild
```

Comment:
why is this a subtype of type bait? think you can just make it a normal bait

Reply:
it was just to piggy-back on its 25 power